src/installer/corehost/cli/common/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+
+project(common)
```

Comment:
Can we rename common here? It seems like it's basically just the hosting PAL and long-file support.

Reply:
Will do.

Reply:
Done; called it `hostmisc` for the lack of a better name.  Let's see what CI has to say this time.